rewrite-csharp/src/main/java/org/openrewrite/csharp/CSharpIsoVisitor.java

```diff
@@ -22,16 +22,10 @@
 
 package org.openrewrite.csharp;
 
-import org.jspecify.annotations.Nullable;
-import org.openrewrite.*;
-import org.openrewrite.internal.ListUtils;
-import org.openrewrite.marker.Markers;
-import org.openrewrite.tree.*;
-import org.openrewrite.java.JavaVisitor;
-import org.openrewrite.java.tree.*;
-import org.openrewrite.csharp.tree.*;
-
-import java.util.List;
+import org.openrewrite.csharp.tree.Cs;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.ControlParentheses;
+import org.openrewrite.java.tree.Parentheses;
```

Comment:
@timtebeek There is that bug again.

Unfortunately we can't easily fix these sources as they are generated and fixing the generator isn't really worth it. What we could do is to have this recipe run after the generator but before we commit.

Reply:
Ah yes I'd mostly been looking to minimize the reported false positives in the daily best practices run: https://app.moderne.io/results/M22gbgF8l
Alternatively we can skip either this project, or changes to generated files. Not sure which route to take, but perhaps indeed not worth investing time in this right now.

Reply:
I did once look into trying to exclude the generated files, but the way we run the recipe makes this kind of hard. This is when I also looked into using Jbang to run recipes, but hit another wall: we can't easily run declarative recipes from an imperative recipe. So I finally just gave up. But regardless of this it would be really nice if we could get this import bug fixed.